core/src/test/scala/unit/kafka/server/DeleteTopicsRequestTest.scala

```diff
@@ -131,12 +131,11 @@ class DeleteTopicsRequestTest extends BaseRequestTest {
   }
 
   private def sendDeleteTopicsRequest(request: DeleteTopicsRequest, socketServer: SocketServer = controllerSocketServer): DeleteTopicsResponse = {
-    val response = connectAndSend(request, ApiKeys.DELETE_TOPICS, socketServer)
-    DeleteTopicsResponse.parse(response, request.version)
+    connectAndReceive[DeleteTopicsResponse](request, destination = socketServer)
   }
 
   private def sendMetadataRequest(request: MetadataRequest): MetadataResponse = {
```

Comment:
Is it even worth having this method? There are a few other examples like that in other classes.